superset/config.py

```diff
@@ -113,6 +113,10 @@
 
 APPLICATION_PREFIX = ''
 
+TICKET_GENERATION_SYSTEM_NAME = 'JIRA'
+TICKET_GENERATION_SYSTEM_ENDPOINT = 'https://guavus-jira.atlassian.net/rest/api/2/issue/'
+TICKET_GENERATION_SYSTEM_API_KEY = 'vOzAgEbwDDQEhqb2idle73FF'
+TICKET_GENERATION_SYSTEM_USER ='arpit.agarwal@guavus.com'
 # Use this flag to enable/disable walkme
```

Comment:
I am exposing these variable values in deployment env variables. You have to overwrite these values in **superset_config.py**.  Please update the code otherwise latest value will not be picked.

Reply:
yes. That is in my task list for today